src/Components/Nav/Components/ProfileDetail.js

```diff
@@ -1,63 +1,48 @@
-import React, { Component } from "react";
-import { Link } from "react-router-dom";
+import React, { useState } from "react";
 import styled from "styled-components";
 import LoginModal from "./LoginModal";
+import SignupModal from "./SignupModal";
 
-export default class ProfileDetail extends Component {
-  constructor() {
-    super();
-    this.state = {
-      isModalActive: false,
-    };
-  }
+export default function ProfileDetail() {
+  const [isLoginActive, setIsLoginActive] = useState(false);
+  const [isSignupActive, setIsSignupActive] = useState(false);
 
-  handleLogin = () => {
-    const { isModalActive } = this.state;
-    this.setState({
-      isModalActive: !isModalActive,
-    });
+  const handleLogin = () => {
+    setIsLoginActive(!isLoginActive);
   };
 
-  handleClose = () => {
-    this.setState({
-      isModalActive: false,
-    });
+  const handleSignup = () => {
+    setIsSignupActive(!isSignupActive);
   };
 
-  render() {
-    const { isModalActive } = this.state;
-    const { handleLogin, handleClose } = this;
-    return (
-      <PoppedProfileSection>
-        <Link to="/" onClick={handleLogin}>
-          로그인
-        </Link>
-        {isModalActive && <LoginModal handleClose={handleClose} />}
-        <Link to="/">회원가입</Link>
-        <Link to="/">숙소 호스트 되기</Link>
-        <Link to="/">체험 호스팅하기</Link>
-        <Link to="/">도움말</Link>
-      </PoppedProfileSection>
-    );
-  }
+  return (
+    <PoppedProfile>
+      <GoTab onClick={() => handleLogin()}>로그인</GoTab>
+      {isLoginActive && <LoginModal handleLogin={handleLogin} />}
+      <GoTab onClick={() => handleSignup()}>회원가입</GoTab>
```

Comment:
```suggestion
      <GoTab onClick={handleLogin}>로그인</GoTab>
      {isLoginActive && <LoginModal handleLogin={handleLogin} />}
      <GoTab onClick={handleSignup}>회원가입</GoTab>
```
즉시 실행함수로 만들었을 때도 역시 정상적으로 동작하나 `{} !== {}` 이듯이 함수 역시 마찬가지입니다. 리액트가 매번 다른 값이 들어온다고 생각하기 때문에 가능하면 handleLogin 함수 자체를 넘겨주는 것이 좋습니다.